### PR TITLE
Add renew certificate github action

### PR DESCRIPTION
**Pull request by smiclea**
Runs the renew-certificate Juju action on the certbot-k8s charm every 2
months and on command.

### CODE REVIEW

File: .github/workflows/renew_certificate.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  build:
+    name: Trigger Juju renew certificate action on certbot-k8s
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install Juju
+        run: |
+          sudo snap install juju --classic
+
+      - name: Install Docker
+        uses: docker-practice/actions-setup-docker@master
```
Comment (claudiubelu) [inline .github/workflows/renew_certificate.yaml:19]: why do you need docker?

File: .github/workflows/renew_certificate.yaml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+        run: |
+          
+          # Controller and model names are the same, "finos-legend"
+          juju switch finos-legend:finos-legend
+                    
+          # Running juju status will show us the current revisions of the charms.
+          juju status --relations
+
+          # Run the renew action on the certbot-k8s charm.
+          juju run-action certbot-k8s/0 renew-certificate
```
Comment (claudiubelu) [inline .github/workflows/renew_certificate.yaml:48]: Hmm, for juju actions, we can have ``juju run-action --wait``, which will wait for the action to complete. Then, we can check if the action was successful or not.

**Review by claudiubelu — COMMENTED** [review]
Depends On: https://github.com/pedroleaoc/certbot-k8s/pull/7

File: .github/workflows/renew_certificate.yaml (hunk 1 above)
Comment (smiclea) [reply .github/workflows/renew_certificate.yaml]: docker dependecy removed

File: .github/workflows/renew_certificate.yaml (hunk 2 above)
Comment (smiclea) [reply .github/workflows/renew_certificate.yaml]: Now using `--wait` and removed the sleep command.

File: .github/workflows/renew_certificate.yaml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+        run: |
+
+          # Controller and model names are the same, "finos-legend"
+          juju switch finos-legend:finos-legend
+                    
+          # Running juju status will show us the current revisions of the charms.
+          juju status --relations
+
+          # Run the renew action on the certbot-k8s charm.
+          juju run-action certbot-k8s/0 renew-certificate --wait
```
Comment (claudiubelu) [inline .github/workflows/renew_certificate.yaml:45]: So, ideally, we would check for the result of the action. According to the documentation (https://juju.is/docs/olm/working-with-actions), there are 5 possible values for the status of an action.

The idea is to fail the github action if the juju action did not complete successfully, which will then show us if something went wrong or not on a simple glance. Currently, this github action would be green even if the juju action failed, which isn't intuitive.

tl;dr: we should check the juju action result. If it's not successful, we should fail the github action / exit 1.

**Comment by claudiubelu** [comment]
The dependency (https://github.com/pedroleaoc/certbot-k8s/pull/7) merged.
